src/components/data/MoveDialog.js

```diff
@@ -0,0 +1,192 @@
+            resourcesMove({ dest, sources });
+        }
+    };
+
+    return (
+        <Formik
+            enableReinitialize
+            initialValues={{ dest: path }}
+            onSubmit={handleMove}
+        >
+            {({ handleSubmit }) => {
+                return (
+                    <Form>
+                        <DEDialog
+                            open={open}
+                            onClose={onClose}
+                            baseId={baseId}
+                            title={t("move")}
+                            actions={
+                                <>
+                                    <Button
+                                        id={build(baseId, ids.CANCEL_BTN)}
+                                        onClick={onClose}
+                                    >
+                                        {i18nCommon("cancel")}
+                                    </Button>
+                                    <Button
+                                        id={build(baseId, ids.MOVE_BTN)}
+                                        color="primary"
+                                        type="submit"
+                                        onClick={handleSubmit}
+                                    >
+                                        {t("move")}
+                                    </Button>
+                                </>
+                            }
+                        >
+                            {status === constants.LOADING && (
+                                <CircularProgress
+                                    size={30}
+                                    thickness={5}
+                                    style={{
+                                        position: "absolute",
+                                        top: "50%",
+                                        left: "50%",
+                                    }}
+                                />
+                            )}
+                            {moveError && (
+                                <ErrorTypographyWithDialog
+                                    errorObject={moveError}
+                                    errorMessage={t("moveRequestFailed")}
+                                />
+                            )}
+                            <Typography
+                                variant="subtitle2"
+                                style={{
+                                    marginTop: theme.spacing(1),
+                                    marginBottom: theme.spacing(1),
+                                }}
+                            >
+                                {t("selectedFilesFolders")}
+                            </Typography>
+                            {(selectedResources === null ||
+                                selectedResources === undefined ||
+                                selectedResources?.length === 0) && (
```

Comment:
I think `selectedResources?.length === 0` also covers the null and undefined check?